redis/CHANGELOG.md

```diff
@@ -83,7 +205,7 @@
 * Revert redis-test versioning changes ([#993](https://github.com/redis-rs/redis-rs/pull/993))
 * Tests: Add retries to test cluster creation ([#994](https://github.com/redis-rs/redis-rs/pull/994))
 * Fix sync cluster behavior with transactions. ([#983](https://github.com/redis-rs/redis-rs/pull/983))
-* Sync Pub/Sub - cache received pub/sub messages. ([#910](https://github.com/redis-rs/redis-rs/pull/910))
+* Sync Pub/Sub * cache received pub/sub messages. ([#910](https://github.com/redis-rs/redis-rs/pull/910))
```

Comment:
Were these (and below) changes intentional?

Reply:
🤦